webtau-config/src/main/java/com/twosigma/webtau/cfg/WebTauConfig.java

```diff
@@ -46,6 +46,11 @@
     private final ConfigValue waitTimeout = declare("waitTimeout", "wait timeout in milliseconds", () -> 5000);
     private final ConfigValue disableFollowingRedirects = declareBoolean("disableRedirects", "disable following of redirects from HTTP calls");
     private final ConfigValue maxRedirects = declare("maxRedirects", "Maximum number of redirects to follow for an HTTP call", () -> 20);
+    private final ConfigValue userAgent = declare("userAgent", "User agent to send on HTTP requests",
+            () -> "webtau/" + getClass().getPackage().getImplementationVersion());
```

Comment:
I think we expose version in some other way

Reply:
I didn't find it but if it is already available I can undo the jar packaging change and switch this line over.